pkgs/by-name/ni/nist-feed/package.nix

```
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, bash
+, jq
+, killall
+, libnotify
+, cron
+}:
+
+stdenv.mkDerivation rec {
```

Considering this is a bash script, it might be fine to build it without the C toolchain

```suggestion
stdenvNoCC.mkDerivation rec {
```

---

@h7x4 probably also `rec` can be removed... I don't see its utilization, right?

---

There's a comment about that further up :)